libs/barista-components/loading-distractor/src/loading-distractor.ts

```diff
@@ -31,7 +36,23 @@ let nextUniqueId = 0;
   preserveWhitespaces: false,
   changeDetection: ChangeDetectionStrategy.OnPush,
 })
-export class DtLoadingDistractor {
+export class DtLoadingDistractor implements AfterViewInit {
   /** @internal The generated label id. */
   _labelId = `"dt-loading-distractor-label-${nextUniqueId++}`;
+
+  /**
+   * @internal
+   * Instance of loading distractor text container
+   */
+  @ViewChild('loadingLabelContainer', { static: false })
+  _loadingLabelContainer: ElementRef;
+
+  constructor(private _zone: NgZone) {}
+
+  ngAfterViewInit(): void {
+    // Focusing on the container allows the screen reader to read the label when the loading distractor is shown
+    this._zone.onStable.pipe(take(1)).subscribe(() => {
+      this._loadingLabelContainer.nativeElement.focus();
```

Comment:
hmm, I'm not sure if this is the right approach. on appearance of the loading distractor this would immediately focus the element. even if the focus was previously somewhere else or the user does not even navigate with a keyboard. 

Imho this will trigger more violations on the a11y tests than it would solve.